### ISSUE TITLE
ManyToMany Relations generate a warning about a table having no fields

### ISSUE THREAD

**Issue by etimberg** [collerek/ormar#295]
**Describe the bug**
When using an M2M field between two tables, Ormar produces a warning that the generated join table has no fields. The exact warning shown below was produced using the example from the [Many To Many docs](https://collerek.github.io/ormar/relations/many-to-many/).

```
WARNING:root:Table posts_categorys had no fields so auto Integer primary key named `id` created.
```

**To Reproduce**
1. Go to https://github.com/etimberg/ormar-test-cases/tree/m2m-warning
2. Clone the repo, check out the `m2m-warning` branch, and follow the readme to start postgres
3. Run `python reproduce.py` in your terminal
4. See a warning 

**Expected behavior**
M2M fields should not generate a warning about a table missing columns

**Versions (please complete the following information):**
 - Database backend used: postgresql
 - Python version: 3.9.6
 - `ormar` version: 0.10.15
 - `pydantic` version: 1.8.2

**Additional context**
This problem can be mitigated by explicitly modelling the the join table and using `through` on the `ManyToMany` field.

**Comment by collerek** [comment]
Yep, that should not be issued for auto generated through models, only if you explicitly declare a through model and don't provide any columns.

Thanks for catching that😊

**Comment by collerek** [comment]
Fixed in #298
